modules/gffread/main.nf

```diff
@@ -15,8 +15,8 @@ process GFFREAD {
     path "versions.yml" , emit: versions
 
     script:
-    def args = task.ext.args ?: ''
-    def prefix = task.ext.suffix ? "${gff.baseName}${task.ext.suffix}" : "${gff.baseName}"
+    def args   = task.ext.args   ?: ''
+    def prefix = task.ext.prefix ?: "${gff.baseName}"
```

Comment:
```suggestion
    def prefix = task.ext.prefix ?: "${gff.baseName}"
```

Is def needed here? Not used in the other modules it looks like

Reply:
In all the other modules (that have already been updated) `prefix` is localized to the script context. It's only unlocalized when used in the output.

Reply:
Ok !